benwbrum/fromthepage
# Show error message if user is logged out for #3182

To test:

- Logged out user scenario
  - Log in and edit a page
  - Leave that tab open and log out in another tab
  - Verify that a warning message appears.
- Bad connection/bad server
  - Log in and edit a page
  - Disconnect your wireless/shut down the server
